### PR TITLE
refactor(qp_interface): prefix package and namespace with autoware

### PR DESCRIPTION
**Pull request by esteve**
## Related links

**Parent Issue:**

- https://github.com/autowarefoundation/autoware/issues/5077
- https://github.com/autowarefoundation/autoware/issues/4569

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by maxime-clem — APPROVED** [review]

**Comment by esteve** [comment]
@maxime-clem thanks

@xmfcx could you merge this? It's got the approval of one of the code owners (@maxime-clem ) Thanks.

**Pull request merged**
